app/wiring/AppComponents.scala

```diff
@@ -21,13 +24,33 @@ class AppComponents(context: Context, stage: String) extends BuiltInComponentsFr
     GoogleAuthConfig(clientId, clientSecret, redirectUrl, domain, antiForgeryChecker = AntiForgeryChecker.borrowSettingsFromPlay(httpConfiguration))
   }
 
+  private val googleGroupChecker = {
+    val s3URI = new AmazonS3URI(configuration.get[String]("googleAuth.serviceAccount.certificateS3Path"))
+    val stream = S3.s3Client
+      .getObject(s3URI.getBucket, s3URI.getKey)
+      .getObjectContent
+
+    val googleServiceAccountCredential = GoogleCredential.fromStream(stream)
+    stream.close()
```

Comment:
will this be closed if there's an exception (and if not does it matter or will the app fail the healthcheck anyway?)

Reply:
ah good point. if anything goes wrong in this class, the app won't start so it will fail healthcheck. we'll get logging on the exception in CloudWatch to diagnose